SimPEG/EM/Static/IP/SurveyIP.py

```diff
@@ -23,3 +22,46 @@ def dpred(self, m=None, f=None):
                 d_\\text{pred} = Pf(m)
         """
         return self.prob.Jvec(m, m, f=f)
+
+
+def from_dc_to_ip_survey(dc_survey, dim="2.5D"):
+    srcList = dc_survey.srcList
```

Comment:
I think we want to copy the object, not assign it. This could lead to some confusing behaviour if you run a dc simulation or inversion in the same place as an IP inversion and then go back to look at or do something with the dc survey since the receiver list is overwritten here. correct me if I am wrong on that @rowanc1 ?

Reply:
I am basically regenerating rxList and srcList here for the IP survey using the DC survey, so they are not connected.